lib/Models/TableColumnStyle.js

```diff
@@ -122,6 +137,12 @@ var TableColumnStyle = function(options) {
     this.featureInfoFields = options.featureInfoFields;
 
     /**
+     * Color for column (css string)
+     * @type {String}
+     */
+    this.color = options.color;
```

Comment:
This only applies to the chart, not to the map, right?  A more descriptive name (`chartLineColor`?) would make it clearer, or at least mention in the doc that it only applies to charting.